### PR TITLE
Add cucumber into Testing section.

### PR DESCRIPTION
**Pull request by MengJueM**
BDD testing framework cucumber is added.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -387,6 +387,7 @@ A curated list of awesome Java frameworks, libraries and software. Inspired by o
 * [Selenium](http://docs.seleniumhq.org/) - Portable software testing framework for web applications.
 * [TestNG](http://testng.org/) - Testing framework.
 * [VisualVM](http://visualvm.java.net/) - Visual interface for detailed information about running applications.
+* [Cucumber](https://github.com/cucumber/cucumber-jvm) - A BDD testing framework that supports the most popular programming languages for the JVM.
```
Comment (akullpp) [inline README.md]: Please care for alphabetical sorting of the entries

**Comment by MengJueM** [comment]
@akullpp the order should be fixed right now

**Comment by akullpp** [comment]
Can you please fix the merge conflict?

**Comment by MengJueM** [comment]
Close this stale PR

**Pull request closed without merging**